scripts/generateThorLongtailTokens/generateThorLongtailTokens.ts

```diff
@@ -73,22 +73,26 @@ export const generateThorLongtailTokens = async () => {
 
   const erc20Tokens = [...ethData.tokens, ...avaxData.tokens]
   const bep20Tokens = bscData.tokens
-  const erc20AssetIds = erc20Tokens.map(token =>
-    toAssetId({
-      chainNamespace: CHAIN_NAMESPACE.Evm,
-      chainReference: String(token.chainId) as ChainReference,
-      assetNamespace: ASSET_NAMESPACE.erc20,
-      assetReference: token.address,
-    }),
-  )
-  const bep20AssetIds = bep20Tokens.map(token =>
-    toAssetId({
-      chainNamespace: CHAIN_NAMESPACE.Evm,
-      chainReference: String(token.chainId) as ChainReference,
-      assetNamespace: ASSET_NAMESPACE.bep20,
-      assetReference: token.address,
-    }),
-  )
+  const erc20AssetIds = erc20Tokens
+    .filter(token => VALID_CHAIN_IDS.eip155.includes(String(token.chainId) as ChainReference))
```

Comment:
not directly related, but was broken since the removal of testnet chainids from caip and wanted to regen everything